submissions/DawoduRahmon/htmlForms/contact.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<head>
+	<meta charset="UTF-8">
+    <meta name="viewport" content="width=div, initial-scale=1.0">
+    <title>Contact-Form</title>
+</head>
+<body>
+<a href="../html/index.html" target="blank">Home</a>
+<div>
+	<h1>Rilogad Limited Nigeria</h1>
+</div>
+<form>
+	<label>First Name <input type="text"/></label>
```

Comment:
This is the right format
<label>First Name</label>
<input type="text">
Kindly update @Dawodurahmonn

Reply:
pls what should i update on these very one ma?

Reply:
Check here [https://www.w3schools.com/html/html_forms_attributes.asp](https://www.w3schools.com/html/html_forms_attributes.asp)
@Dawodurahmonn